### ISSUE TITLE
VS Code 1.44.2: Doesn't work

### ISSUE THREAD

**Issue by GHSoCoder** [helsmy/autohotkey-debug-adaptor#2]
Hi helsmy,

great idea really, many thanks for all your effort! But doesn't work with my VS Code 1.44.2.
I have a simple AHK test script. Right from the green 'play' button there are two items in the drop-down list:
1. Ask for file name
2. Add configuration...

I set one breakpoint. When I press the green 'play' button the debug buttons are popping up for some seconds. nothing happens.
Must I change something in the launch.json?

Greetings
GHSoCoder

**Comment by helsmy** [comment]
If you want to just start debugging, just pressing f5 is enough. f5 will debug the script open in workspace. launch.json is a specific setting for each project, which is not necessary. Without it, you script will start debugging in default setting. My default setting of launch.json has a confusing name which I had changed in last commit.

**Comment by GHSoCoder** [comment]
No, it doesn't work! When I press F5, a sort of green bar (just a short one pixel line) is moving in the RUN panel. Then as mentioned the debug buttons are popping up for some seconds (3s).
On the left side bar is an icon with an arrow and bug. First a number (1) is shown, then an clock (in blue). After the mentioned 3s the clock disapears together with the debug buttons.
My simple test script, breakpoint at line 1:
`x := 10`
`MsgBox, % "x: " x`

**Comment by helsmy** [comment]
it should work. Maybe, you could check you ahk executable path, and make sure it is `C:\Program Files\AutoHotkey\AutoHotkey.exe`. Otherwise, you must set AhkExecutable to your specific path in launch.json. If all of them do not work, use this dev-debug version to run it again and send me your ahkdbg.log file( in `youruserfile\.vscode\extensions\helsmy.autohotkey-debug-100.2.2\ahkdbg`)

[autohotkey-debug-100.2.2.zip](https://github.com/helsmy/autohotkey-debug-adaptor/files/4523818/autohotkey-debug-100.2.2.zip)

**Comment by GHSoCoder** [comment]
Basically it's working. The problem was my non-standard path to AutoHotkey.exe. But be aware that the backslash `\` must be doubled in file launch. json, like `C:\\...\\...\\AutoHotkey.exe`. The template item should show that. 
I have overlooked that setting in launch.json. Strange was that after restarting Visual Studio Code, then debugging _but before_ setting the path I got now an error message from AutoHotkey that AutoHotkey.exe couldn't be found. I have never gotten that error message before.

New problems:
1. My variable x is not shown under VARIABLES -> Local.
2. How can I add that variable `x` as WATCH?

Suggestions:
1. Setting in launch. json that an AutoHotkey GUI is set to topmost automatically. So it is not hidden from Visual Studio Code.
2. Also an setting for auto `<CR><LF>` after an OutputDebug. Really great is that OutputDebug-messages are shown in the DEBUG CONSOLE!

Hint:
1. With setting "program": "${workspaceRoot}/${fileBasename}" scripts (e. g. for tests) are not found in sub folders.
